### PR TITLE
[@types/bip21] Add optional 'options' return parameter to decode function

### PR DESCRIPTION
**Pull request by verbiricha**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [2.0.0 changelog](https://github.com/bitcoinjs/bip21/blob/master/CHANGELOG.md)
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/bip21/index.d.ts [hunk 1]
```diff
@@ -1,11 +1,11 @@
-// Type definitions for bip21 v1.1.2
+// Type definitions for bip21 v2.0.0
 // Project: https://github.com/bitcoinjs/bip21
 // Definitions by: Stefan Huber <https://github.com/stefanhuber>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 declare namespace bip21 {
-    export function decode(uri:string) : {address:string,amount?:number | undefined};
-    export function encode(address:string,options?:any) : string;
+    export function decode(uri: string): { address: string; amount?: number | undefined; options: object | undefined };
```
Comment (peterblazejewicz) [inline types/bip21/index.d.ts:7]: this looks wrong, IMO only `address` and `amount` are returned in the object:
https://github.com/bitcoinjs/bip21/blob/master/index.js#L24

File: types/bip21/index.d.ts [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
+    export function encode(address: string, options?: any): string;
```
Comment (peterblazejewicz) [inline types/bip21/index.d.ts:8]: this also looks like missing `urnScheme` parameter:
https://github.com/bitcoinjs/bip21/blob/master/index.js#L27

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@verbiricha thanks!

**Comment by verbiricha** [comment]
Ready to merge

**Pull request merged**